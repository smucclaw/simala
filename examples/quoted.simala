#eval
let `this is a variable` = 3 in
let `this is another variable` = 22 in
let `atoms can be quoted too` = '`see here` in
let `a record` = { `field 1` = `atoms can be quoted too` } in

if `a record` . `field 1` == '`see there`
  then `this is a variable` + `this is another variable`
  else `this is a variable` * `this is another variable`
