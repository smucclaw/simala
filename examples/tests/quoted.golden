`this is a variable` = 3
`this is another variable` = 22
`atoms can be quoted too` = '`see here`
`a record` = {`field 1` = '`see here`}
66
